plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -206,29 +207,42 @@ def is_struct_included(struct1, struct2, exclude=None):
         description:
             Return True if all element of dict 1 are present in dict 2, return false otherwise.
     """
-    if isinstance(struct1, list) and isinstance(struct2, list):
-        for item1 in struct1:
-            if isinstance(item1, (list, dict)):
-                for item2 in struct2:
-                    if not is_struct_included(item1, item2, exclude):
+    try:
```

Comment:
I think this diff would probably be a lot more readable if the `try` would stay in the place where it is actually needed, and wouldn't cover the whole function.